dev/scripts/generate-release-tarballs.sh

```diff
@@ -29,11 +40,16 @@ function main {
   for hadoop_profile in "${HADOOP_PROFILES[@]}"; do
     echo "Building tarball for ${hadoop_profile}"
     "${GENERATE_TARBALL_SCRIPT}" --deleteUnrevisioned --hadoopProfile ${hadoop_profile}
-    local tarball="${SCRIPT_DIR}/tarballs/$(ls -tr ${SCRIPT_DIR}/tarballs | tail -1)"
-    md5 "${tarball}" > "${tarball}.md5"
-    if [[ "$(dirname ${tarball})" != "${build_directory}" ]]; then
-      cp "${tarball}" "${build_directory}"
-      cp "${tarball}.md5" "${build_directory}"
+    local tarball="$(ls -tr ${SCRIPT_DIR}/tarballs | tail -1)"
+    local full_path_tarball="${SCRIPT_DIR}/tarballs/${tarball}"
+    if [[ "$(dirname ${full_path_tarball})" != "${build_directory}" ]]; then
+      cp "${full_path_tarball}" "${build_directory}"
+      if [[ ! -z ${md5_cmd} ]]; then
+        cd "${build_directory}"
```

Comment:
We would need to use `full_path_tarball` but that won't get us the md5 we want, I've left a comment clarifying this.